tests/test_dataset.py

```diff
@@ -179,25 +174,14 @@ def test_raise_exceptions(self):
         with pytest.raises(SequencingRunError):
             self.dataset.raise_exceptions()
 
-    def test_resolve_pipeline_and_toolset(self):
-        self.dataset._pipeline_instruction = Mock(
-            return_value={'toolset_type': 'a_toolset', 'toolset_version': 3, 'name': 'qc'}
-        )
-        with patch(ppath + 'toolset') as mocked_toolset:
-            self.dataset.resolve_pipeline_and_toolset()
-
-        mocked_toolset.select_type.assert_called_with('a_toolset')
-        mocked_toolset.select_version.assert_called_with(3)
-        assert self.dataset.pipeline.name == 'qc'
-
-    @patch(ppath + 'toolset', new=Toolset())
-    def test_pipeline_instruction(self):
-        self.dataset._default_pipeline = Mock(return_value='qc')
-        assert self.dataset._pipeline_instruction() == {
-            'name': 'qc',
-            'toolset_type': 'non_human_sample_processing',
-            'toolset_version': 0
-        }
+
+class _TestDataset(Dataset):
+    type = 'None'
+    endpoint = 'None'
+    id_field = 'None'
+
+    def _is_ready(self):
+        pass
```

Comment:
This was removed in master, but seems to have crept back in during the merge. Can this be reverted, along with the removal of `patched_is_ready`?